cinema/models.py

```diff
@@ -50,8 +50,8 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
```

Comment:
The `genres` and `actors` fields in the `Movie` model are correctly set as `ManyToManyField` with `blank=True`, which aligns with the migration changes. This allows movies to be created without initially assigning genres or actors.